sdcs-site/.angular-cli.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Are you sure all resources in sdcs-site are constantly needed and used?

Furthermore, cannot we rely that the person who clones the repo will read the docs, run the `npm` commands and all these files will be locally built? What I mean is - isn't committing all of these resources in sdcs-site the same like committing the maven target folder?

After we clarify this, then I will have a deeper look at the frontend sources.